service/push-consumer/service.go

```diff
@@ -13,13 +20,15 @@ type Service struct {
 	health   *health.Core
 	config   *Config
 	registry registry.Registry
+	NodeID   string
 }
 
 // NewService creates an instance of new push consumer service
 func NewService(ctx context.Context, config *Config) *Service {
 	return &Service{
 		ctx:    ctx,
 		config: config,
+		NodeID: uuid.New().String(),
```

Comment:
Should we take the k8s stateful set as the node id?

Reply:
Yes, i remember that discussion, we should do that, just keeping some random id for start, will add todo